tests/spread/integration/iputils-ping/task.yaml

```diff
@@ -0,0 +1,8 @@
+summary: Integration tests for iputils-ping
+
+execute: |
+  # Chisel a minimum number of slices to give us a runnable system that we can
+  # test in.
+  rootfs="$(install-slices iputils-ping_bins)"
+
+  chroot "${rootfs}/" ping -c 4 8.8.8.8
```

Comment:
Le'ts just make this execute `ping -h`, as there's no guarantee that where the unit tests are executed has actually network access to ping a public IP address.